agent/stats/queue.go

```diff
@@ -179,7 +212,8 @@ func (queue *Queue) enoughDatapointsInBuffer() bool {
 
 // getCWStatsSet gets the stats set for either CPU or Memory based on the
 // function pointer.
-func (queue *Queue) getCWStatsSet(f getUsageFunc) (*ecstcs.CWStatsSet, error) {
+// stats are float64 type
```

Comment:
nit: add the period at the end of sentence?

Reply:
`stats are float64 type.` ? sure I'll make this change.

Reply:
nit: I think there are other places also missing the period for the comments before function. Since it can be used to generate go doc. (I am not sure whether people will do this). It will be nice if our comments are well formatted. Other than this, LGTM.

Reply:
I ended up removing this particular comment.  When we have time, it wouldn't be a terrible thing to audit our comments agent-wide for style issues.